middle_end/flambda2/simplify/simplify_apply_cont_expr.ml

```diff
@@ -79,6 +79,28 @@ let rebuild_apply_cont apply_cont ~args ~rewrite_id uacc ~after_rebuild =
       let apply_cont =
         Simplify_common.clear_demoted_trap_action uacc apply_cont
       in
+      let apply_cont =
+        if AC.is_raise apply_cont then
+          match AC.args apply_cont with
+          | [] -> assert false
+          | first_arg :: other_args ->
```

Comment:
Maybe we this should be named something like `exn_arg` or `exn_bucket` to explicit the fact that it is the exception being raised ?